scripts/check-updates.ts

```diff
@@ -0,0 +1,44 @@
+import fs from 'node:fs/promises';
+import { $ } from 'zx';
+import pRetry from 'p-retry';
+import { NPM_TAGS, type NpmTag, readVersionFile } from './utils';
+
+async function main() {
+  for (let tag of NPM_TAGS) {
+    await checkUpdate(tag);
+  }
+}
+
+async function checkUpdate(tag: NpmTag) {
+  console.log(`Checking update for npm tag: ${tag}`);
+  const currentVersion = await readVersionFile(`prisma-${tag}`);
+  const npmVersion = await getNpmVersion(tag);
+  console.log(`Current version ${currentVersion}`);
+  console.log(`npm version ${npmVersion}`);
+  if (currentVersion !== npmVersion) {
+    console.log('Update needed');
+    const ghOut = process.env.GITHUB_OUTPUT;
+    if (ghOut) {
+      await fs.appendFile(ghOut, `${tag}=${npmVersion}\n`);
+    }
+  } else {
+    console.log('Up to date');
+  }
+}
+
+async function getNpmVersion(tag: NpmTag) {
+  const content = await pRetry(
+    () => $`yarn npm info prisma@${tag} --json -f version`,
```

Comment:
```suggestion
    () => $`npm info prisma@${tag} --json -f version`,
```
No need for yarn right?

Reply:
What's `-f` already?

Reply:
This repo uses yarn exclusively. I briefly tried to convert to `pnpm` and result of it was pretty much that `react-native` and `pnpm` really don't like each other. 

`yarn npm` is not forwarding to npm, it's yarn subcommand for interacting with registry.
`-f` brings back only requested fields from a registry.